### ISSUE TITLE
Unexpected error in SpaceAfterPropertyColon

### ISSUE THREAD

**Issue by honzabilek4** [brigade/scss-lint#795]
Hey, 
I've run into this issue while editing my .scss files. Sadly I don't remeber the exact change which broke the code. Can you please inspect this error? Here's the console output:

```
SCSSLint::Linter::SpaceAfterPropertyColon raised unexpected error linting file sass/_mobile.scss: 'undefined method `+' for nil:NilClass'
C:/Ruby22/lib/ruby/gems/2.2.0/gems/scss_lint-0.48.0/lib/scss_lint/linter/space_after_property_colon.rb:80:in `whitespace_after_colon'
C:/Ruby22/lib/ruby/gems/2.2.0/gems/scss_lint-0.48.0/lib/scss_lint/linter/space_after_property_colon.rb:16:in `visit_prop'
C:/Ruby22/lib/ruby/gems/2.2.0/gems/sass-3.4.22/lib/sass/tree/visitors/base.rb:36:in `visit'
C:/Ruby22/lib/ruby/gems/2.2.0/gems/scss_lint-0.48.0/lib/scss_lint/linter.rb:138:in `visit'
C:/Ruby22/lib/ruby/gems/2.2.0/gems/scss_lint-0.48.0/lib/scss_lint/linter.rb:149:in `block in visit_children'
C:/Ruby22/lib/ruby/gems/2.2.0/gems/scss_lint-0.48.0/lib/scss_lint/linter.rb:147:in `each'
C:/Ruby22/lib/ruby/gems/2.2.0/gems/scss_lint-0.48.0/lib/scss_lint/linter.rb:147:in `visit_children'
C:/Ruby22/lib/ruby/gems/2.2.0/gems/sass-3.4.22/lib/sass/tree/visitors/base.rb:36:in `block in visit'
C:/Ruby22/lib/ruby/gems/2.2.0/gems/scss_lint-0.48.0/lib/scss_lint/linter/space_after_property_colon.rb:12:in `visit_rule'
C:/Ruby22/lib/ruby/gems/2.2.0/gems/sass-3.4.22/lib/sass/tree/visitors/base.rb:36:in `visit'
C:/Ruby22/lib/ruby/gems/2.2.0/gems/scss_lint-0.48.0/lib/scss_lint/linter.rb:138:in `visit'
C:/Ruby22/lib/ruby/gems/2.2.0/gems/scss_lint-0.48.0/lib/scss_lint/linter.rb:149:in `block in visit_children'
C:/Ruby22/lib/ruby/gems/2.2.0/gems/scss_lint-0.48.0/lib/scss_lint/linter.rb:147:in `each'
C:/Ruby22/lib/ruby/gems/2.2.0/gems/scss_lint-0.48.0/lib/scss_lint/linter.rb:147:in `visit_children'
C:/Ruby22/lib/ruby/gems/2.2.0/gems/sass-3.4.22/lib/sass/tree/visitors/base.rb:36:in `block in visit'
C:/Ruby22/lib/ruby/gems/2.2.0/gems/scss_lint-0.48.0/lib/scss_lint/linter/space_after_property_colon.rb:12:in `visit_rule'
C:/Ruby22/lib/ruby/gems/2.2.0/gems/sass-3.4.22/lib/sass/tree/visitors/base.rb:36:in `visit'
C:/Ruby22/lib/ruby/gems/2.2.0/gems/scss_lint-0.48.0/lib/scss_lint/linter.rb:138:in `visit'
C:/Ruby22/lib/ruby/gems/2.2.0/gems/scss_lint-0.48.0/lib/scss_lint/linter.rb:149:in `block in visit_children'
C:/Ruby22/lib/ruby/gems/2.2.0/gems/scss_lint-0.48.0/lib/scss_lint/linter.rb:147:in `each'
C:/Ruby22/lib/ruby/gems/2.2.0/gems/scss_lint-0.48.0/lib/scss_lint/linter.rb:147:in `visit_children'
C:/Ruby22/lib/ruby/gems/2.2.0/gems/sass-3.4.22/lib/sass/tree/visitors/base.rb:38:in `visit'
C:/Ruby22/lib/ruby/gems/2.2.0/gems/scss_lint-0.48.0/lib/scss_lint/linter.rb:138:in `visit'
C:/Ruby22/lib/ruby/gems/2.2.0/gems/scss_lint-0.48.0/lib/scss_lint/linter.rb:41:in `run'
C:/Ruby22/lib/ruby/gems/2.2.0/gems/scss_lint-0.48.0/lib/scss_lint/runner.rb:53:in `run_linter'
C:/Ruby22/lib/ruby/gems/2.2.0/gems/scss_lint-0.48.0/lib/scss_lint/runner.rb:35:in `block in find_lints'
C:/Ruby22/lib/ruby/gems/2.2.0/gems/scss_lint-0.48.0/lib/scss_lint/runner.rb:33:in `each'
C:/Ruby22/lib/ruby/gems/2.2.0/gems/scss_lint-0.48.0/lib/scss_lint/runner.rb:33:in `find_lints'
C:/Ruby22/lib/ruby/gems/2.2.0/gems/scss_lint-0.48.0/lib/scss_lint/runner.rb:19:in `block in run'
C:/Ruby22/lib/ruby/gems/2.2.0/gems/scss_lint-0.48.0/lib/scss_lint/runner.rb:18:in `each'
C:/Ruby22/lib/ruby/gems/2.2.0/gems/scss_lint-0.48.0/lib/scss_lint/runner.rb:18:in `run'
C:/Ruby22/lib/ruby/gems/2.2.0/gems/scss_lint-0.48.0/lib/scss_lint/cli.rb:74:in `scan_for_lints'
C:/Ruby22/lib/ruby/gems/2.2.0/gems/scss_lint-0.48.0/lib/scss_lint/cli.rb:59:in `act_on_options'
C:/Ruby22/lib/ruby/gems/2.2.0/gems/scss_lint-0.48.0/lib/scss_lint/cli.rb:33:in `run'
C:/Ruby22/lib/ruby/gems/2.2.0/gems/scss_lint-0.48.0/bin/scss-lint:7:in `<top (required)>'
C:/Ruby22/bin/scss-lint:23:in `load'
C:/Ruby22/bin/scss-lint:23:in `<main>'
Report this bug at https://github.com/brigade/scss-lint/issues

To help fix this issue, please include:
- The above stack trace
- SCSS-Lint version: 0.48.0
- Sass version: 3.4.22
- Ruby version: 2.2.3
```

**Comment by sds** [comment]
Hey @honzabilek4,

Thanks for the report. Sadly, this stack trace is hard to interpret without a reproducible case. A clear example is necessary so that we treat the underlying problem and not simply mask this particular case.

If you manage to reproduce feel free to reopen with the example. Thanks!
